crates/flipperzero/Cargo.toml

```diff
@@ -41,6 +41,7 @@ embedded-hal-0 = { package = "embedded-hal", version = "0.2.7", features = ["unp
 
 # Docs
 document-features = { workspace = true, optional = true }
+uom = { version = "0.36.0", features = ["si", "u32"], default-features = false }
```

Comment:
I suggest that we add all new dependencies to workspace root and reference them here via `workspace = true`

Also note, that this part of `Cargo.toml` is for docs-related dependencies (see comment two lines above)

Reply:
Makes sense to me, I will hold off on this change for now since I need to properly figure out how to use uom crate since at present my use of it utilizes dynamic dispatch. I also want to read up on cargo workspaces since I have not used them before.